code/modules/events/anomaly/anomaly_placer.dm

```diff
@@ -65,6 +65,7 @@
 		/area/station/solars,
 		/area/station/holodeck,
 		/area/station/maintenance,
+		/area/station/science/ordnance/bomb,
```

Comment:
can you alphabetize this list? there's a few extensions in vscode that do this for you. thanks

Reply:
yep sorry will do!

Reply:
no worries this sort of thing isn't really written down anywhere it's just a nicety